core/base/src/constants/circle.ts

```diff
@@ -25,7 +25,8 @@ const usdcContracts = [[
     ["Sonic",     "0x29219dd400f2Bf60E5a23d13Be72B486D4038894"],
     ["Linea",     "0x176211869cA2b568f2A7D4EE941E073a821EE1ff"],
     ["Worldchain","0x79A02482A880bCE3F13e09Da970dC34db4CD24d1"],
-    ["Seievm",    "0xe15fC38F6D8c56aF07bbCBe3BAf5708A2Bf42392"]
+    ["Seievm",    "0xe15fC38F6D8c56aF07bbCBe3BAf5708A2Bf42392"],
+    ["HyperEVM",  "0xb88339cb7199b77e23db6e890353e22632ba630f"]
```

Comment:
per @kev1n-peters should be the checksummed address `0xb88339CB7199b77E23DB6E890353E22632Ba630f`